Update scenario management overview from develop (#571)
Avaiga/taipy-doc: docs/tutorials/complete_applications/scenario_management_overview/index.md

!!! important "Supported Python versions"

    Taipy requires **Python 3.8** or newer.

Welcome to the **Tutorial** guide for scenario management.

> Taipy facilitates pipeline orchestration. There are several good reasons to use the Taipy backend:

Reviewer: ```suggestion
Taipy simplifies pipeline orchestration, and there are several reasons to utilize the Taipy backend:
```

- Taipy efficiently manages the execution of your functions/pipelines.

- Taipy manages data sources and monitors KPIs.

> - Taipy makes it simple to oversee multiple pipelines and end-user scenarios, especially useful in Machine Learning or Mathematical optimization contexts.

Reviewer: ```suggestion
- Taipy makes it easy to manage multiple pipelines and end-user scenarios, which is particularly valuable in Machine Learning or Mathematical optimization scenarios.
```

    [Python installation guide](http://docs.python-guide.org/en/latest/starting/installation/)
    can guide you through the process.

## Taipy Studio

> There are two ways to configure Taipy backend (Taipy Core), either by Python code or with Taipy Studio. We strongly recommend using Taipy Studio. 

Reviewer: ```suggestion
You have two options for configuring Taipy backend (Taipy Core): either through Python code or by using Taipy Studio. We highly recommend using Taipy Studio.
```

> Taipy Studio is a VS Code extension that provides a graphical editor to describe pipelines. The configuration of Taipy Core can be done more easily and quickly through Taipy Studio.

Reviewer: ```suggestion
Taipy Studio is a Visual Studio Code extension that offers a graphical editor for defining pipelines. Configuring Taipy Core becomes much simpler and faster when using Taipy Studio.
```

> So, without further delay, let's begin to code!

Reviewer: ```suggestion
Now, without any more delay, let's get started with the coding!
```
